src/users/service.js

```diff
@@ -14,6 +14,8 @@ const UserService = {
     user,
   }).then(() => UserService.login(user.email, user.password)),
 
+  userProfileData: () => Vue.http.post(`${Vue.config.apiServer}/api/2.0/profiles/user-profile-data`, {}),
```

Comment:
pretty sure this endpoint shoudln't be called with empty body but rather with a query